libraries/cyclestreets-core/build.gradle

```diff
@@ -1,13 +1,11 @@
-
-
 configurations {
   compile.exclude module: 'commons-logging'
+  compile.exclude module: 'httpclient'
```

Comment:
This exclusion is necessary as it seems the latest version of `httpmime` pulls in `httpcomponents` transitively, which clashes with what Android includes natively.
>WARNING: Dependency org.apache.httpcomponents:httpclient:4.5.2 is ignored for <build target> as it may be conflicting with the internal version provided by Android.
         In case of problem, please repackage it with jarjar to change the class packages